# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/moz.build
# Commit: e25543fae7ee
# Full Hash: e25543fae7ee03ac57b9cfbfeb2418c9e2f490fc
# Author: Martin Robinson <mrobinson@igalia.com>
# Date: 2022-06-23 03:54:50
# Regressor Bug: 1463600
# File Overlap Count: 1
# Description:
#   Bug 1463600 - Implement CSS 'contain: style' r=emilio
#   
#   Add an implementation of CSS `contain: style`. This introduces two new
#   data structures, the ContainStyleScope and ContainStyleScopeManager.
#   
# ==============================================================================

diff -r 4230bdcf07ae -r e25543fae7ee layout/base/moz.build
--- a/layout/base/moz.build	Wed Jun 22 16:15:57 2022 +0000
+++ b/layout/base/moz.build	Wed Jun 22 16:16:59 2022 +0000
@@ -72,6 +72,7 @@
 EXPORTS.mozilla += [
     "AccessibleCaretEventHub.h",
     "ArenaObjectID.h",
+    "ContainStyleScopeManager.h",
     "DisplayPortUtils.h",
     "GeckoMVMContext.h",
     "GeometryUtils.h",
@@ -101,6 +102,7 @@
     "AccessibleCaret.cpp",
     "AccessibleCaretEventHub.cpp",
     "AccessibleCaretManager.cpp",
+    "ContainStyleScopeManager.cpp",
     "DisplayPortUtils.cpp",
     "GeckoMVMContext.cpp",
     "GeometryUtils.cpp",